kolibri_instant_schools_plugin/assets/src/views/reset-password-page/status.vue

```diff
@@ -51,12 +51,12 @@
       explanation() {
         switch (this.status) {
           case STATES.CHECKING_TOKEN:
-            return this.$tr('confirmingToken');
+            return this.$tr('confirmingLink');
           case STATES.PASSWORD_CHANGED:
             return this.$tr('passwordSuccessfullyChanged');
           case STATES.LINK_EXPIRED:
             return this.$tr('needToRequestNewToken');
-          case OTHER_ERROR:
+          case STATES.OTHER_ERROR:
```

Comment:
Nice catch

Reply:
flying blind eslint-less in this project!